src/V4Router.sol

```diff
@@ -114,7 +120,7 @@ abstract contract V4Router is IV4Router, BaseActionsRouter, DeltaResolver {
                 params.hookData
             )
         ).toUint128();
-        if (amountIn > params.amountInMaximum) revert TooMuchRequested();
+        if (amountIn > params.amountInMaximum) revert V4TooMuchRequested();
```

Comment:
Renaming because in UR we sensibly have `V2TooMuchRequested` and `V3TooMuchRequested`. So this will go nicely